Alicerce/Sources/Extensions/UICollectionView+ReusableView.swift

```diff
@@ -19,11 +19,12 @@ public extension UICollectionView {
         return cell
     }
     
-    public func register<T: UICollectionViewCell>(_ cellType: T.Type) where T: ViewCellProtocol {
+    public func register<T: UICollectionViewCell>(_ cellType: T.Type) where T: ReusableView {
         register(cellType, forCellWithReuseIdentifier: cellType.reuseIdentifier)
     }
     
-    public func register<T: AnyObject>(_ viewType: T.Type, forSupplementaryViewOfKind kind: String) where T: ViewCellProtocol {
+    public func register<T: UICollectionReusableView>(_ viewType: T.Type, forSupplementaryViewOfKind kind: String)
+        where T: ReusableView {
```

Comment:
I tend to line up `where` clauses at the same indentation level as the `func` keyword. However, it's a matter of preference.